### DISCUSSION TITLE
Toast on Windows platform

### DISCUSSION THREAD

**Discussion by sbll885** [CommunityToolkit/Maui#1725] [Q&A] [answered]
I'm trying to use a toas on windows platform (MAUI application). No exception is thrown but toast is not shown. I tried to create a new application according [instructions](https://learn.microsoft.com/en-us/dotnet/communitytoolkit/maui/alerts/toast?tabs=android) or use toast [sample](https://github.com/CommunityToolkit/Maui/tree/main/samples) given by this project. 
The question is it a bug or Toast is limited for specific platforms ?
The closes discussion I [found](https://github.com/CommunityToolkit/Maui/discussions/1178) doesn't explain how to handle the issue. Thanks.

**Comment by VladislavAntonyuk** [comment, marked as answer]
Check “Do not disturb” is not active and your app allows notifications

**Reply by sbll885** [reply]
Thanks!

**Comment by CliffAgius** [comment]
Closed as answered.
